cookiecutter-frontend-app/{{cookiecutter.repo_name}}/.env.development

```diff
@@ -1,7 +1,7 @@
 NODE_ENV='development'
-PORT=8080
+PORT={{cookiecutter.port}}
```

Comment:
And then the rest of the changes are just interpolating variables into files, their names and contents.